web/src/features/dataset/pages/ViewOneDatasetPage.tsx

```
@@ -46,7 +46,7 @@ const ViewOneDatasetPage = () => {
   });
 
   const dataset = datasetQuery.data;
-  const isManager = Boolean(datasetQuery.data?.managerIds.includes(currentUser!.id));
+  const isManager = Boolean(dataset?.managerIds.includes(currentUser!.id));
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)